src/Config.php

```diff
@@ -51,6 +30,15 @@ public function __construct($config)
     /** @var string Password to use for authentication */
     public $password;
 
+    /** @var bool Whether to use SSL */
+    public $useSsl;
+
+    /** @var array Dynamic options for SQL SSL connections, etc. */
+    public $dynamicOptions = [];
```

Comment:
Should not be public and renamed to something like `extraSettings` or `specificSettings`.